test/doc_logs_bang_test.rb

```diff
@@ -0,0 +1,28 @@
+require File.expand_path(File.dirname(__FILE__) + "/test_helper")
+
+class DocLogsBangTest < MiniTest::Test
+  describe "doc_logs!" do
+    before do
+      DocRaptor.api_key "something something"
+    end
+
+    describe "with bogus arguments" do
+      it "should raise an error if something other than an options hash is passed in" do
+        assert_raises(ArgumentError) {DocRaptor.doc_logs!(true)}
+        assert_raises(ArgumentError) {DocRaptor.doc_logs!(nil)}
+      end
+    end
+
+    describe "with good arguments" do
+      it "should give me a valid response" do
+        stub_http_response_with("simple_list_docs.xml", :get)
```

Comment:
probably need to add a new fixture that is like 'simple_list_doc_logs.xml' for this case.